src/model/transformations/collected-variable.js

```diff
@@ -85,6 +89,10 @@ export function remoteToStore(
           datatype.mayears = matches_maximum[0] == 0 ? '' : matches_maximum[0];
           datatype.mamonths = matches_maximum[1] == 0 ? '' : matches_maximum[1];
         }
+        if (format !== undefined && format === 'HH:CH') {
```

Comment:
Why do you need to test undefined && HH:CH ? The second test is not enough ?

Reply:
The test was already part of existing code, but I believe the second test is enough, yes. With and without the test the result is the same, so I have removed it.